packages/celotool/src/lib/blockscout.ts

```diff
@@ -215,3 +217,41 @@ export async function switchIngressService(celoEnv: string, ingressName: string)
    -p='[{"op": "replace", "path": "/spec/rules/0/http/paths/0/backend/serviceName", "value":"${ingressName}-web"}]'`
   await execCmdWithExitOnFailure(command)
 }
+
+export async function accessSecretVersion() {
+  try {
+    const client = new SecretManagerServiceClient()
+    const projectId = fetchEnv(envVar.GRAFANA_CLOUD_PROJECT_ID)
+    const secretName = fetchEnv(envVar.GRAFANA_CLOUD_SECRET_NAME)
+    const secretVersion = fetchEnv(envVar.GRAFANA_CLOUD_SECRET_VERSION)
+    const [version] = await client.accessSecretVersion({
+      name: `projects/${projectId}/secrets/${secretName}/versions/${secretVersion}`,
+    })
+
+    const privateKey = JSON.parse(version?.payload?.data?.toString()!).grafana_api_token
+
+    if (!privateKey) {
+      throw new Error('Key is empty or undefined')
+    }
+
+    return privateKey
+  } catch (error) {
+    console.info('Error retrieving key')
+  }
+}
+
+export async function createGrafanaTagAnnotation(celoEnv: string, tag: string, suffix: string) {
+  const token = await accessSecretVersion()
+  await fetch('https://clabs.grafana.net/api/annotations', {
```

Comment:
Should be the url parameter read from some config repository? Maybe adding a new field to the secret?